diff --git a/setup.py b/setup.py
index 15c3353..418b99e 100644
--- setup.py
+++ setup.py
@@ -18,9 +18,9 @@ setup(name='poppy-creature',
       version=version(),
       packages=find_packages(),

-      install_requires=['pypot[http-server]>=2.9.0', 'bottle'],
+      install_requires=['pypot', 'bottle'],

-      setup_requires=['setuptools_git >= 0.3', ],
+      setup_requires=['setuptools_git', ],

       include_package_data=True,
       exclude_package_data={'': ['README.md', '.gitignore']},
